sink/sqs.go

```diff
@@ -143,10 +144,21 @@ func (s *SQSSink) write() {
 				id = id + 1
 			}
 
-			_, err := s.sqs.SendMessageBatch(&sqs.SendMessageBatchInput{
-				Entries:  entries,
-				QueueUrl: aws.String(s.queueName),
-			})
+			err := s.sendBatch(entries)
+			if err != nil && strings.Contains(err.Error(), "AWS.SimpleQueueService.BatchRequestTooLong") && len(entries) > 3 {
```

Comment:
what if the length is 2, but still too big? shouldn't we retry those one at a time?

Reply:
maybe... it is quite unlikely, though

Reply:
just thinking we could accidentally, and silently drop messages this way - would be super hard to debug in 9 months :P